preprocessors/togeojson-gpx.preprocessor.js

```diff
@@ -102,6 +102,14 @@ module.exports = function(infile, outdirectory, callback) {
       });
     });
 
+    var archivedOriginal = path.join(outdirectory, '/archived.gpx');
+    digest(infile, function(err, callback) {
+      fs.writeFile(archivedOriginal, fs.readFile(infile), function(err) {
```

Comment:
Thanks for pointing that out. I changed this step to use fs.readFileSync, as well as in the tests. Console log'ing showed the proper values, and the equivalence of the initial file to the archived file.